LICENSE

```diff
@@ -0,0 +1,78 @@
+﻿--[[ ZeroBrane Studio License ]]-------------------------------------------
```

Comment:
I think we need a line / paragraph at the top that says something like "Our original code is licensed on a per-file basis. Other project's licenses are included below for reference."

Reply:
I just used that